UPGRADING.md

```diff
@@ -1,5 +1,144 @@
-# Upgrading Gaia
+# Upgrade Gaia from v14.1.0 to v14.2.0
 
-This guide provides instructions for upgrading to specific versions of Gaia.
+## This is a security upgrade. IT IS CONSENSUS BREAKING, so please apply the fix only on height 19460500.
```

Comment:
Is the height agreed? Above it says TBA.

Reply:
yeah, it was agreed. By above, you mean release notes?